crate/operator/__init__.py

```diff
@@ -1,18 +1,23 @@
 # CrateDB Kubernetes Operator
-# Copyright (C) 2020 Crate.IO GmbH
```

Comment:
Taken from https://github.com/crate/crate-python

Do we need the Copyright statement with the year?